clients/telemetry-pubsub/src/main/java/org/eclipse/hono/client/telemetry/pubsub/PubSubBasedDownstreamSender.java

```diff
@@ -199,11 +192,11 @@ private Map<String, Object> addDefaults(
                 .map(HashMap::new)
                 .orElseGet(HashMap::new);
 
-        messageProperties.put(PUBSUB_PROPERTY_DEVICE_ID, device.getDeviceId());
+        messageProperties.put(PUBSUB_DOWNSTREAM_PROPERTY_DEVICE_ID, device.getDeviceId());
         messageProperties.put(MessageHelper.APP_PROPERTY_QOS, qos.ordinal());
-        messageProperties.put(PUBSUB_PROPERTY_TENANT_ID, tenant.getTenantId());
-        messageProperties.put(PUBSUB_PROPERTY_DEVICE_REGISTRY_ID, tenant.getTenantId());
-        messageProperties.put(PUBSUB_PROPERTY_PROJECT_ID, projectId);
+        messageProperties.put(PUBSUB_DOWNSTREAM_PROPERTY_TENANT_ID, tenant.getTenantId());
```

Comment:
shouldn't this be `MessageHelper.APP_PROPERTY_TENANT_ID`?

Reply:
see above comment